pkgs/applications/editors/vim/plugins/nvim-treesitter/overrides.nix

```diff
@@ -27,6 +27,25 @@ let
       })
     generatedDerivations;
 
+  grammarToPlugin = grammar:
+    let
+      name = lib.pipe grammar [
+        lib.getName
+
+        # added in buildGrammar
+        (lib.removeSuffix "-grammar")
+
+        # grammars from tree-sitter.builtGrammars
+        (lib.removePrefix "tree-sitter-")
+        (lib.replaceStrings [ "-" ] [ "_" ])
+      ];
+    in
+
+    runCommand "nvim-treesitter-${name}-grammar" { } ''
```

Comment:
I would prefer `runCommand "nvim-treesitter-grammar-${name}" { } ''` but if that breaks some coherency later on let's forget about it

Reply:
done